staging/src/k8s.io/apiserver/pkg/util/version/version.go

```diff
@@ -167,15 +167,3 @@ func DefaultKubeEffectiveVersion() MutableEffectiveVersion {
 	binaryVersion := version.MustParse(baseversion.DefaultKubeBinaryVersion).WithInfo(baseversion.Get())
 	return newEffectiveVersion(binaryVersion, false)
 }
-
-// ValidateKubeEffectiveVersion validates the EmulationVersion is equal to the binary version at 1.31 for kube components.
-// TODO: remove in 1.32
-// emulationVersion is introduced in 1.31, so it is only allowed to be equal to the binary version at 1.31.
-func ValidateKubeEffectiveVersion(effectiveVersion EffectiveVersion) error {
```

Comment:
should we still be validating that this is within a valid range? e.g. >= 1.31 for now, or maybe N-3 in the future, or something like that?

Reply:
Created https://github.com/kubernetes/kubernetes/issues/127514 to track this